templates/jws50-tomcat9-https-s2i.json

```diff
@@ -0,0 +1,442 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-rh-tomcat",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "tags": "tomcat,tomcat8,java,jboss,hidden",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss Web Server 5.0 Apache Tomcat 9 (with https)",
+            "description": "An example JBoss Web Server application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Web Server 5.0 Apache Tomcat 9 based application, including a build configuration, application deployment configuration, and secure communication using https.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-web-server/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+
+        },
+        "name": "jws50-tomcat9-https-s2i"
+    },
+    "labels": {
+        "template": "jws50-tomcat9-https-s2i",
+        "jws": "1.0"
```

Comment:
We should make these all consistent, so jws or jws50.  Sorry I wasn't clear in my previous comment.  Same should apply to the image streams.

Reply:
@rcernich Made the appropriate changes, I hope I understood your comment correctly.